lxd/device/nic_routed.go

```diff
@@ -205,6 +208,69 @@ func (d *nicRouted) validateEnvironment() error {
 	return nil
 }
 
+func (d *nicRouted) checkIPAvailability(parent string) error {
+	ipv4Addr, ok := d.config["ipv4.address"]
```

Comment:
The ipv*.address fields are a comma separated list of ips so please use SplitNTrimSpace function

Reply:
I've done this. However this does mean that there will be a 100ms delay for each IP address in the list. Perhaps we can reduce the timeout? I'm not sure what a sensible value will be as this will be system dependent.

Reply:
ARP and NDP checks are now concurrent.